sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/GeneralScalarExpression.java

```diff
@@ -20,22 +20,42 @@
 import java.io.Serializable;
 
 import org.apache.spark.annotation.Evolving;
+import org.apache.spark.sql.connector.util.V2ExpressionSQLBuilder;
 
 /**
- * The general SQL string corresponding to expression.
+ * The general representation of SQL scalar expressions, which contains the upper-cased
+ * expression name and all the children expressions.
  *
```

Comment:
Why do we remove the classdoc? Without the doc, how can data source developers match different expressions?

Reply:
It seems JDK11 and JKD17 maven build exceeds 5 hours. I just check the issue.
I will restore the classdoc later.